docker/testnet/docker-compose.yaml

```diff
@@ -99,6 +99,11 @@ services:
       - AMARU_NETWORK=testnet_42
     volumes:
       - a1-state:/srv/amaru
+    depends_on:
+      cardano-loader:
```

Comment:
This is not needed as it's implied in the `x-amaru` fragment: the `Amaru-loader` depends on `Cardano-loader`.